SPECS/pg_partman.spec

```diff
@@ -0,0 +1,108 @@
+/usr/pgsql-9.4/bin/reapply_constraints.py
+/usr/pgsql-9.4/bin/reapply_foreign_keys.py
+/usr/pgsql-9.4/bin/reapply_indexes.py
+/usr/pgsql-9.4/bin/undo_partition.py
+/usr/pgsql-9.4/lib/pg_partman_bgw.so
+/usr/pgsql-9.4/share/extension/pg_partman--0.1.0--0.1.1.sql
+/usr/pgsql-9.4/share/extension/pg_partman--0.1.1--0.1.2.sql
+/usr/pgsql-9.4/share/extension/pg_partman--0.1.2--0.2.0.sql
+/usr/pgsql-9.4/share/extension/pg_partman--0.2.0--0.3.0.sql
+/usr/pgsql-9.4/share/extension/pg_partman--0.3.0--0.3.1.sql
+/usr/pgsql-9.4/share/extension/pg_partman--0.3.1--0.3.2.sql
+/usr/pgsql-9.4/share/extension/pg_partman--0.3.2--0.4.0.sql
+/usr/pgsql-9.4/share/extension/pg_partman--0.4.0--0.4.1.sql
+/usr/pgsql-9.4/share/extension/pg_partman--0.4.1--0.4.2.sql
+/usr/pgsql-9.4/share/extension/pg_partman--0.4.2--1.0.0.sql
+/usr/pgsql-9.4/share/extension/pg_partman--1.0.0--1.1.0.sql
+/usr/pgsql-9.4/share/extension/pg_partman--1.1.0--1.2.0.sql
+/usr/pgsql-9.4/share/extension/pg_partman--1.2.0--1.3.0.sql
+/usr/pgsql-9.4/share/extension/pg_partman--1.3.0--1.4.0.sql
+/usr/pgsql-9.4/share/extension/pg_partman--1.4.0--1.4.1.sql
+/usr/pgsql-9.4/share/extension/pg_partman--1.4.1--1.4.2.sql
+/usr/pgsql-9.4/share/extension/pg_partman--1.4.2--1.4.3.sql
+/usr/pgsql-9.4/share/extension/pg_partman--1.4.3--1.4.4.sql
+/usr/pgsql-9.4/share/extension/pg_partman--1.4.4--1.4.5.sql
+/usr/pgsql-9.4/share/extension/pg_partman--1.4.5--1.5.0.sql
+/usr/pgsql-9.4/share/extension/pg_partman--1.5.0--1.5.1.sql
+/usr/pgsql-9.4/share/extension/pg_partman--1.5.1--1.6.0.sql
+/usr/pgsql-9.4/share/extension/pg_partman--1.6.0--1.6.1.sql
+/usr/pgsql-9.4/share/extension/pg_partman--1.6.1--1.7.0.sql
+/usr/pgsql-9.4/share/extension/pg_partman--1.7.0--1.7.1.sql
+/usr/pgsql-9.4/share/extension/pg_partman--1.7.1--1.7.2.sql
+/usr/pgsql-9.4/share/extension/pg_partman--1.7.2--1.8.0.sql
+/usr/pgsql-9.4/share/extension/pg_partman--1.8.0--1.8.1.sql
+/usr/pgsql-9.4/share/extension/pg_partman--1.8.1--1.8.2.sql
+/usr/pgsql-9.4/share/extension/pg_partman--1.8.2--1.8.3.sql
+/usr/pgsql-9.4/share/extension/pg_partman--1.8.3--1.8.4.sql
+/usr/pgsql-9.4/share/extension/pg_partman--1.8.4--1.8.5.sql
+/usr/pgsql-9.4/share/extension/pg_partman--1.8.5--1.8.6.sql
+/usr/pgsql-9.4/share/extension/pg_partman--1.8.6--1.8.7.sql
+/usr/pgsql-9.4/share/extension/pg_partman--1.8.7--1.8.8.sql
+/usr/pgsql-9.4/share/extension/pg_partman--1.8.7--2.0.0.sql
+/usr/pgsql-9.4/share/extension/pg_partman--1.8.8--2.0.0.sql
+/usr/pgsql-9.4/share/extension/pg_partman--2.0.0--2.1.0.sql
+/usr/pgsql-9.4/share/extension/pg_partman--2.1.0--2.2.0.sql
+/usr/pgsql-9.4/share/extension/pg_partman--2.2.0--2.2.1.sql
+/usr/pgsql-9.4/share/extension/pg_partman--2.2.1--2.2.2.sql
+/usr/pgsql-9.4/share/extension/pg_partman--2.2.2--2.2.3.sql
+/usr/pgsql-9.4/share/extension/pg_partman--2.2.3.sql
+/usr/pgsql-9.4/share/extension/pg_partman.control
+/usr/share/doc/pgsql/extension/pg_partman.md
+/usr/share/doc/pgsql/extension/pg_partman_howto.md
```

Comment:
@olivermel Please add the %{pg_dir} variable to all file paths.  the /share/doc paths should be %{pg_dir}/doc/extension/...